backend/qemu.pm

```diff
@@ -216,10 +216,36 @@ sub start_qemu {
         die "no Qemu/KVM found\n" unless $qemubin;
     }
 
-    if ($vars->{BIOS} && !-e '/usr/share/qemu/' . $vars->{BIOS}) {
+    if ($vars->{UEFI}) {
+        # XXX: compat with old deployment
+        $vars->{BIOS} //= $vars->{UEFI_BIOS};
+    }
+
+    if ($vars->{BIOS} && $vars->{BIOS} !~ /^\//) {
+        # Non-absolute paths are assumed relative to /usr/share/qemu
+        $vars->{BIOS} == '/usr/share/qemu/' . $vars->{BIOS};
+    }
```

Comment:
yes, sorry, I had some kinda virus yesterday - did this in another place too, but caught that one as I hit it in testing...I didn't test this bit.